aws-predictions/src/androidTest/java/com/amplifyframework/predictions/aws/AWSPredictionsInterpretTest.java

```diff
@@ -168,17 +165,24 @@ public void testNegativeSentimentDetection() throws Exception {
      */
     @Test
     public void testKeyPhraseDetection() throws Exception {
+        final String sampleText = "My mama always said life was like a box of chocolates.";
+
         // Interpret sample text and assert non-null result
         InterpretResult result = predictions.interpret(
-                Assets.readAsString("sample-text-en.txt"),
+                sampleText,
                 InterpretOptions.defaults()
         );
         assertNotNull(result);
 
-        // Assert detected key phrases are not empty
-        List<KeyPhrase> keyPhrases = result.getKeyPhrases();
-        assertNotNull(keyPhrases);
-        assertFalse(keyPhrases.isEmpty());
+        // Assert key phrase detection
+        List<KeyPhrase> actual = result.getKeyPhrases();
+        List<String> expected = Arrays.asList(
+                "My mama",
+                "life",
+                "a box",
```

Comment:
Idle thought: Does Comprehend provide a guarantee around what it considers a "key phrase?" For example, will we regret coupling our test to whatever Comprehend's models return for a given sentence. For example, in the future, could Comprehend omit articles from key phrases and just return the noun? Perhaps the answer is, "idk? We'd just update the test."

Reply:
That's my approach for now haha. We'll keep this the way is until the tests break -> update the tests. If it becomes a semi-frequent enough issue, we will revisit to see if we can have a "snapshot" of the comprehend model